front-end/src/app/shared/services/login.service.ts

```diff
@@ -66,12 +66,10 @@ export class LoginService {
     if (this.userLoginData && this.userLoginData.token) { // Non-login.gov auth
       this.store.dispatch(userLoggedOutAction());
     } else {
-      this.apiService.postAbsoluteUrl(`${environment.loginDotGovLogoutUrl}`, null).pipe(
-        tap(() => {
-          this.clearUserLoggedInCookies();
-          this.store.dispatch(userLoggedOutAction());
-        })
-      ).subscribe(() => undefined);
+      this.cookieService.delete('csrftoken');
+      window.location.href = environment.loginDotGovLogoutUrl || "";
```

Comment:
Seems like this is the fix 👍 .  As for the code before, `pipe(tap(...))` [is for side effects](https://rxjs.dev/api/operators/tap) and is redundant to the subscribe below where line 71 and 72 would be more appropriate